nuxeo-coldstorage/src/main/java/org/nuxeo/coldstorage/service/ColdStorageServiceImpl.java

```diff
@@ -478,14 +479,14 @@ public void propagateMoveToColdStorage(CoreSession session, String blobDigest) {
 
         BulkService bulkService = Framework.getService(BulkService.class);
         String username = SecurityConstants.SYSTEM_USERNAME;
-        String commandId = bulkService.submit(
+        String commandId = bulkService.submitTransactional(
                 new BulkCommand.Builder(PropagateMoveToColdStorageContentAction.ACTION_NAME, query, username).build());
 
         BulkStatus status = bulkService.getStatus(commandId);
         if (status == null) {
```

Comment:
the status is never null, if the command id is not found, you have a BulkStatus with an UNKNOWN state